example/src/main/java/org/apache/rocketmq/example/openmessaging/SimpleProducer.java

```diff
@@ -40,15 +39,15 @@ public static void main(String[] args) {
         System.out.printf("Producer startup OK%n");
 
         {
-            Message message = producer.createBytesMessage("OMS_HELLO_TOPIC", "OMS_HELLO_BODY".getBytes(Charset.forName("UTF-8")));
```

Comment:
Why not use StandardCharsets.UTF_8 ? IMO, some os may use gbk as default decoder.

Reply:
I don't think it's necessary to specify the encode charset for message body.
As long as one encode and decode the message with the same charset, he shall get the correct message.